Gems/PhysX/Code/Tests/EditorMeshColliderComponentTests.cpp

```diff
@@ -34,6 +34,56 @@
 
 namespace PhysXEditorTests
 {
+    static bool MeshColliderHasOnePhysicsAssetShapeType(PhysX::MeshColliderComponent* meshColliderComponent)
+    {
+        if (!meshColliderComponent)
+        {
+            return false;
+        }
```

Comment:
If the tests depend on these, is it wise to throw a return here instead an assert since the test is going to fail anyways? 

If you're planning on handling these possibilities in tests, using an enumerated return code pattern and failing tests that way would provide greater value.

Reply:
NVM, I misread what was going on here after going back and forth between Python & C++ all day.